app/models/solidus_subscriptions/failure_dispatcher.rb

```diff
@@ -3,7 +3,7 @@
 module SolidusSubscriptions
   class FailureDispatcher < Dispatcher
     def dispatch
-      order.touch :completed_at
+      # order.touch :completed_at
```

Comment:
Could we go ahead and change this to `order.touch :updated_at`? Seems like it would make more sense and allow us to remove these comments :man_shrugging:

Reply:
I think the following call to `cancel!` will update that attribute.